src/kOS/Persistence/PersistenceExtensions.cs

```diff
@@ -151,9 +151,5 @@ private static FileContent Decode(string input)
             }
         }
 
-        private static string EncodeBase64(string input)
-        {
-            return PersistenceUtilities.EncodeBase64(Encoding.ASCII.GetBytes(input));
-        }
```

Comment:
This no longer needed to exist since the one place it was being called got deleted, and I don't want anyone ever using this, since it encodes with vanilla ASCII instead of with UTF-8.